src/components/Page/AdjacentPages.jsx

```diff
@@ -15,7 +15,7 @@ AdjacentPages.propTypes = {
 export default function AdjacentPages({ previous, next }) {
   if (!previous && !next) return null;
   return (
-    <div className="adjacent-links">
+    <div className="adjacent-links print:hidden">
```

Comment:
No need to show those links for printed page.

Reply:
Now they are easy to miss, I think they should have better visibility. Maybe we can just move them `10px` up?

Reply:
Could you can share a screenshot to show why do you think they are easy to miss?

Reply:
Because they are not at bottom of the page, users might have to scroll long if  docs are large for example - 
https://webpack-js-org-git-fork-chenxsan-feature-im-b01bac-webpack-docs.vercel.app/configuration/dev-server/

But I see gatsby is also using the same place for these links -
https://www.gatsbyjs.com/docs/how-to/querying-data/use-static-query/

So I think it will be okay for us too. I don't have a strong opiniom here.

Reply:
Users care about the content most, not those links. They are for contributors, not that important thing on the page as long as we have it presented somewhere.